dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/workflow/postgres/xmlworkflow/V7.0_2018.04.03__upgrade_workflow_policy.sql

```diff
@@ -0,0 +1,27 @@
+--
+-- The contents of this file are subject to the license and copyright
+-- detailed in the LICENSE and NOTICE files at the root of the source
+-- tree and available online at
+--
+-- http://www.dspace.org/license/
+--
+
+-- UPDATE policies for claimtasks
+-- Item
+UPDATE RESOURCEPOLICY SET rptype = 'TYPE_WORKFLOW' WHERE dspace_object in (SELECT cwf_workflowitem.item_id FROM cwf_workflowitem INNER JOIN cwf_claimtask ON cwf_workflowitem.workflowitem_id = cwf_claimtask.workflowitem_id JOIN item ON cwf_workflowitem.item_id = item.uuid) AND eperson_id not in (SELECT item.submitter_id FROM cwf_workflowitem JOIN item ON cwf_workflowitem.item_id = item.uuid);
```

Comment:
does this replace item WRITE/READ permissions with workflowitem WRITE/READ permissions to ensure items cannot be edited directly?

Reply:
No, this is just to "mark" the policies already in places and necessary for the XmlWorkflow as TYPE_WORKFLOW to be able to ignore them in the upload panel to determine which access conditions has been applied by the user. In a edge configuration you could have groups used for the workflow also used as a kind of access condition (i.e. Librarians)